dask_bigquery/core.py

```diff
@@ -2,6 +2,7 @@
 
 from contextlib import contextmanager
 from functools import partial
+from typing import List
```

Comment:
Nitpick: I'm by no means a type annotation expert, but from reviewing https://github.com/dask/distributed/pull/5328, I think the recommend approach for a list would be to add `from __future__ import annotations` and then just use the built-in `list` in the type annotation (i.e. `list[str]`). This isn't meant to be a blocking comment as what you have here is valid. Just meant as an FYI

Reply:
oh yeah, good call..I'm used to `list[]` for 3.9 but forgot about the `__future__` option